Extensions/xkit_main.js

```diff
@@ -166,7 +166,7 @@ XKit.extensions.xkit_main = new Object({
 		}
 
 		try {
-			eval(xkit_main.script);
+			eval(xkit_main.script + "\n//# sourceURL=xkit/"+extension_id+".js");
```

Comment:
Also same as above. It's worth looking at whether these reevaluations can be skipped in a future issue (I removed the first one of these at some point with no ill effects, but put it back in because paranoia)